### DISCUSSION TITLE
Stash hotkeys for navigation

### DISCUSSION THREAD

**Discussion by ikonomov** [diasurgical/DevilutionX#6297] [Ideas]
It would be really nice if we can bind hotkeys for navigating the stash.  Arrow keys, Page Up/Down, mouse's back/forward and the scroll wheel are all currently not in use when the stash is open.

**Comment by Gerwin2k** [comment]
In the Win32 v1.5.0 build I just tested: Mouse scroll wheel can be used to navigate the stash pages. Keyboard arrows up/down keys can be used to navigate the stash pages.
